### ISSUE TITLE
Animate view that is not paired to previous view controller

### ISSUE THREAD

**Issue by m-revetria** [CosmicMind/Motion#6]
Hello! 

How can a view that doesn't have a `motionIdentifier` or simply doesn't match any identifier from the previous view controller to be animated at the same time the presentation animation happens?

I tried to add animations to this view using `motionAnimations` but they seem that are not executed.

I additionally tried to use `view.motion(...)` to animate this view in `viewWillAppear` but if I don''t use `delay` I cannot see the animations. I achieved what you can see in next gif adding a delay to the animations of this view after view controller is pushed and before it's popped. As you can notice the white view with the game's description appears after the transition has finished.

![motion_animation_issue](https://cloud.githubusercontent.com/assets/4791678/23135816/cd9da208-f778-11e6-99a5-79e5bda16cdf.gif)

Example code:

```swift
    /********* Pushed view controller's code *********/
    override func viewDidLoad() {
        super.viewDidLoad()
        // Set initial state for animatable views
        backgroundImage.motion(.duration(0.0), .translateX(-view.frame.width))
        tableView.motion(.duration(0.0), .translateY(view.frame.height))
    }

    override func viewWillAppear(_ animated: Bool) {
        super.viewWillAppear(animated)

        let duration = 0.3
        let delay = 0.3
        let timing = MotionAnimation.timingFunction(.easeInEaseOut)

        backgroundImage.motion(.delay(delay), .duration(duration), .translateX(0), timing)
        tableView.motion(.delay(delay), .duration(duration), .translateY(0), timing)
    }

    @IBAction func backButtonDidTouch(_ sender: Any) {
        let duration = 0.3
        let timing = MotionAnimation.timingFunction(.easeInEaseOut)

        backgroundImage.motion([.duration(duration), .translateX(-view.frame.width), timing])
        tableView.motion([.duration(duration), .translateY(view.frame.height), timing])

        DispatchQueue.main.asyncAfter(deadline: .now() + duration) { [unowned self] in
            _ = self.navigationController?.popViewController(animated: true)
        }
    }
```

**Comment by daniel-jonathan** [comment]
There is a delegation method available. I am going to test it and add a sample with the updates I am making today.

**Comment by daniel-jonathan** [comment]
This is now available in the next release. Thank you!

**Comment by daniel-jonathan** [comment]
The fix for this issue is in [Motion 1.1.0](https://github.com/CosmicMind/Motion/releases/tag/1.1.0).
